src/content/zh-tw/resources/translations.md

```diff
@@ -0,0 +1,70 @@
+project_path: /web/_project.yaml
+book_path: /web/resources/_book.yaml
+description: "歡迎您將Web Fundamentals翻譯去其他的語言。"
+
+{# wf_updated_on: 2017-10-10 #}
+{# wf_published_on: 2016-09-13 #}
+
+# 翻譯 {: .page-title }
+
+<!--div class="attempt-right">
+  <figure>
+    <img src="/web/images/gitlocalize_image0.png">
+  </figure>
+</div-->
+
+我們現在在嘗試一個翻譯工具叫做[GitLocalize](https://gitlocalize.com/)。您可以跟著以下的步驟來翻譯Web Fundementals。
+
+1. 去[GitLocalize中的Web Fundamentals軟件庫(repository)](https://gitlocalize.com/repo/107).
+2. 使用您的GitHub帳戶開啟新的GitLocalize帳戶。
+3. 尋找您想要翻譯的文章。
+4. 開始翻譯。
+5. 當翻譯完畢，放送您的翻譯以進行檢查。
+6. 檢查完畢的文章將會被語言管理員作為Pull Request發送到GitHub。
+
+您可以從GitLocalize的[幫助頁面](https://docs.gitlocalize.com/)學到GitLocalize是如何操作的。
+
+如果您遇到任何問題或者有功能請求，您可以在[GitLocalize的issue tracker](https://github.com/gitlocalize/feedback/issues)提供建議。
+
+與其他翻譯貢獻者聊天，您可以[註冊ChromiumDev的Slack](https://chromiumdev-slack.herokuapp.com/)並加入#l10n頻道。
```

Comment:
Need to keep the `Note:` styles, as its a special case for our markdown processor.

Reply:
I don't see the note bit?